### PR TITLE
Localize SSH provider project.

### PR DESCRIPTION
**Pull request by richardstanton**
Update the SSH provider project to allow microbuild localization.

 @pieandcakes @gregg-miskelly @jacdavis @chuckries @paulmaybee

### CODE REVIEW

File: build/miengine.targets [hunk 1]
```diff
@@ -26,6 +26,27 @@
     </Task>
   </UsingTask>
 
+    <!-- Localize the output assembly if requested -->
+  <ItemGroup Condition=" '$(LocalizeOutputAssembly)'=='true' ">
+    <FilesToLocalize Include="$(TargetPath)">
+      <TranslationFile>$(RepositoryRoot)LOC\LCL\{Lang}\$(TargetFileName).lcl</TranslationFile>
```
Comment (pieandcakes) [inline build/miengine.targets]: If we care about cross compat if we need to be able to build on Linux in the future, these should match the case of the folders in the repo

**Comment by pieandcakes** [comment]
@richardstanton left you one comment, but otherwise LGTM

:shipit:

**Comment by gregg-miskelly** [comment]
@richardstanton I believe you also need to update a .nuspec to include the localized dlls in MIEngine's internal nuspec.

**Comment by pieandcakes** [comment]
@richardstanton Did you try and build the nuget package locally? After looking at the MIEngine.mdd.nuspec it looks like its pulling in stuff from drop\<configuration>\loc so as long as your file is there it should be in the package.

**Pull request merged**